CRM/Report/Form/Contribute/Bookkeeping.php

```diff
@@ -388,6 +388,7 @@ public function from() {
               INNER JOIN civicrm_contribution {$this->_aliases['civicrm_contribution']}
                     ON {$this->_aliases['civicrm_contact']}.id = {$this->_aliases['civicrm_contribution']}.contact_id AND
                          {$this->_aliases['civicrm_contribution']}.is_test = 0
+                         {$this->_aliases['civicrm_contribution']}.is_template = 0
```

Comment:
@jaapjansma Is this missing an "AND"?

Reply:
well spotted! Thanks! I will update this PR

Reply:
I have updated this